custom_metrics/third-parties.js

```diff
@@ -0,0 +1,34 @@
+[third-parties]
+/**
+ * Detect the presence of third party libraries.
+ *
+ * Look for common library aliases on the global scope.
+ * If available, detect libraries' versions.
+ *
+ * Outputs JSON-serialized list of library-version pairs.
+ *     e.g.: `["a@1.0","b","c@2.2"]`
+ */
+var thirdParties = [];
+var name, version;
+
+function addThirdParty(name, version) {
+	if (!name) {
+		return;
+	}
+
+	if (version) {
+		name += '@' + version;
+	}
+
+	thirdParties.push(name);
+}
+
+// jQuery
+var jQuery = window.jQuery || window.$ || window.$jq || window.$j;
+name = jQuery && jQuery.fn && 'jQuery';
+version = name && jQuery.fn.jquery
```

Comment:
wishlist: link to the library source where this is defined, so we have a reference / can track changes more easily.